### DISCUSSION TITLE
cache tab doesn't seem to show cache hits for collection cache

### DISCUSSION THREAD

**Discussion by mackross** [julienbourdeau/debugbar#44] [Ideas]
If I use `<%= render partial: 'path_to_partial', collection: @collection, cached: true %>`, the cache hits are shown in the logs, but they're not shown in the debugbar.

**Comment by julienbourdeau** [comment]
Hey @mackross, sorry for the delay, I had some personal situation where I was away from computer for a few months but I'm back!

My first guest is that Rails doesn't dispatch the same Event. I 'll have a look but I should probably add a `views` panel for erb stuff. At work, I always work with React environment that's why I was not in a rush to add it ^^

**Comment by julienbourdeau** [comment]
I'll convert this issue to "discussion" (mostly to try it).
